src/theory/fp/theory_fp_rewriter.cpp

```diff
@@ -410,6 +411,67 @@ namespace constantFold {
     }
   }
 
+  RewriteResponse minTotal (TNode node, bool) {
+    Assert(node.getKind() == kind::FLOATINGPOINT_MIN_TOTAL);
+    Assert(node.getNumChildren() == 3);
+
+    FloatingPoint arg1(node[0].getConst<FloatingPoint>());
+    FloatingPoint arg2(node[1].getConst<FloatingPoint>());
+
+    Assert(arg1.t == arg2.t);
+
+    // Can be called with the third argument non-constant
+    if (node[2].getMetaKind() == kind::metakind::CONSTANT) {
+      BitVector arg3(node[2].getConst<BitVector>());
+
+      FloatingPoint folded(arg1.minTotal(arg2, arg3.isBitSet(0)));
+      Node lit = NodeManager::currentNM()->mkConst(folded);
+      return RewriteResponse(REWRITE_DONE, lit);
+
+    } else {
+      FloatingPoint::PartialFloatingPoint res(arg1.min(arg2));
+
+      if (res.second) {
+	Node lit = NodeManager::currentNM()->mkConst(res.first);
+	return RewriteResponse(REWRITE_DONE, lit);
+      } else {
+	// Can't constant fold the underspecified case
+	return RewriteResponse(REWRITE_DONE, node);
```

Comment:
Just out of curiosity. Why not do something similar to what you do with the rounding modes where you list all possible outcomes? 

https://github.com/martin-cs/CVC4/blob/floating-point-symfpu/src/theory/fp/theory_fp_rewriter.cpp#L1269-L1282